module/VuFind/src/VuFind/Controller/CombinedController.php

```diff
@@ -225,6 +225,17 @@ public function resultsAction()
         $settings = $this->serviceLocator->get(\VuFind\Config\PluginManager::class)
             ->get('config');
 
+        // Identify if any modules use include_recommendations_side.
+        $columnSideRecommendations = [];
+        $recommendationManager = $this->serviceLocator->get(\VuFind\Recommend\PluginManager::class);
+        foreach ($config as $subconfig) {
+            foreach ($subconfig['include_recommendations_side'] ?? [] as $recommendation) {
```

Comment:
The comment says that include_recommendations_side can be set to true to use existing recommendation configuration, but this code seems incompatible with that assertion. Should we revise the comment or change the code?

Reply:
Crap good catch.  I revised the comment.  I don't think it would be good practice to blindly use the usual set of recommendations for any given module, especially given that we tell people not to duplicate them.  It would be
too easy to add something to say EDS.ini thinking about that specific search and not realize that it would break
something in combined mode.